### ISSUE TITLE
[visualization] Change color of arms depending on angle

### ISSUE THREAD

**Issue by RaquelMotzfeldtTirach** [frdedynamics/appirh_project_helse_frde#1]
## Summary
We want the users (patient and care giver) to easily see the level of movement the patient has. 
This will start very simply with just the arms, and also monitoring the angle between arm and core-body.

### Basic example
The main idea is to have the arm change color depending on the said angle. If this is too complicated, maybe we overlay something else that shows that data and with colors

### Details
We've been don't to not use red/orange/green color-set.

**Comment by RaquelMotzfeldtTirach** [comment]
I believe this might be our solution: https://wiki.ros.org/rviz/DisplayTypes/Marker

**Comment by fjnn** [comment]
No, unfortunately this is not applicable on xacro/urdf. You can create and modify additional markers in different colors and shapes. You can spawn markers on the link (upper arm link etc.) and move them attached to the respective link like a mask. It is how I implemented in the HRC project. It works but not perfect and a bit cumbersome to set up.

We can think of this as plan-B or we might reconsider the visual design - do we really need 'the arm' to change color?

The ideal would be if we find a way to change xacro material color, let say a launch argument which is linked to a rosparam, and modify it on the go.

**Comment by RaquelMotzfeldtTirach** [comment]
Yes I agree, the markers were supposed to be a plan B, but it seems like changing the color of a xacro material on the fly is not possible. That's why I'm looking into plan B
